diff -urN kadu.org/kadu/kadu.cpp kadu/kadu/kadu.cpp
--- kadu.org/kadu/kadu.cpp	Mon May 12 22:53:26 2003
+++ kadu/kadu/kadu.cpp	Mon May 12 22:54:29 2003
@@ -1669,20 +1669,27 @@
 		readevent->start(10000, TRUE);
 		}
 
-	if (e->type == GG_EVENT_CONN_FAILED) {
+	if (e->type == GG_EVENT_CONN_FAILED || e->type == GG_EVENT_DISCONNECT) {
 		char error[512];
 		snprintf(error, sizeof(error), "Kadu::eventHandler(): Unable to connect, the following error has occured:\n%s\nKadu::eventHandler(): Keep trying to connect?\n", strerror(errno));
-		trayicon->showErrorHint(i18n("Connection failed"));
-		kdebug("Kadu::eventHandler(): Connection failed\n");
 		kdebug(error);
 		
-		disconnectNetwork();	
-		setCurrentStatus(GG_STATUS_NOT_AVAIL);
 
 		if (autohammer)
 			setStatus(loginparams.status & (~GG_STATUS_FRIENDS_MASK));
 		}
 
+	if (e->type == GG_EVENT_DISCONNECT) {
+		trayicon->showErrorHint(i18n("Disconnection has been occured"));
+		kdebug("Kadu::eventHandler(): Disconnection has been occured\n");
+
+		disconnectNetwork();
+		setCurrentStatus(GG_STATUS_NOT_AVAIL);
+
+		if (e->type == GG_EVENT_DISCONNECT)
+			autohammer = false;
+		}
+
 	if (socket_active) {
 		if (sess->state == GG_STATE_IDLE && userlist_sent) {
 			char error[512];
