diff -urN kadu-0.1.9.org/kadu/kadu.h kadu-0.1.9/kadu/kadu.h
--- kadu-0.1.9.org/kadu/kadu.h	Thu Nov 29 15:55:09 2001
+++ kadu-0.1.9/kadu/kadu.h	Thu Nov 29 16:01:11 2001
@@ -53,8 +53,6 @@
 #include <config.h>
 #endif
 
-#include <libgg.h>
-
 #include <kapp.h>
 #include <klocale.h>
 #include <qevent.h>
@@ -99,6 +97,7 @@
 #include <qmenubar.h>
 
 #include <pthread.h>
+#include <time.h>
 #include <sys/time.h>
 #include <ctype.h>
 #include <errno.h>
@@ -111,6 +110,8 @@
 #include <sys/wait.h>
 #include <sys/socket.h>
 
+#include <libgg.h>
+
 #include "history.h"
 #include "pixmaps.h"
 
