diff -urN kadu.orig/kadu/modules.cpp kadu/kadu/modules.cpp
--- kadu.orig/kadu/modules.cpp	2004-06-30 13:58:45.000000000 +0200
+++ kadu/kadu/modules.cpp	2004-08-26 01:16:50.790962312 +0200
@@ -297,7 +297,7 @@
 QTranslator* ModulesManager::loadModuleTranslation(const QString& module_name)
 {
 	QTranslator* translator=new QTranslator(0);
-	if(translator->load(dataPath("kadu/modules/translations/"+module_name+QString("_") + config_file.readEntry("General", "Language", QTextCodec::locale())), "."))
+	if(translator->load(("/usr/lib/kadu/modules/translations/"+module_name+QString("_") + config_file.readEntry("General", "Language", QTextCodec::locale())), "."))
 	{
 		qApp->installTranslator(translator);
 		return translator;
@@ -361,7 +361,7 @@
 
 QStringList ModulesManager::installedModules() const
 {
-	QDir dir(dataPath("kadu/modules"),"*.so");
+	QDir dir(("/usr/lib/kadu/modules"),"*.so");
 	dir.setFilter(QDir::Files);
 	QStringList installed;
 	for(unsigned int i=0; i<dir.count(); i++)
@@ -411,7 +411,7 @@
 		return true;
 	}
 
-	ConfigFile desc_file(dataPath("kadu/modules/"+module_name+".desc"));
+	ConfigFile desc_file(("/usr/lib/kadu/modules/"+module_name+".desc"));
 
 	QString lang=config_file.readEntry("General", "Language", "en");
 
@@ -529,7 +529,7 @@
 	}
 	else
 	{
-		m.lib=new Library(dataPath("kadu/modules/"+module_name+".so"));
+		m.lib=new Library(("/usr/lib/kadu/modules/"+module_name+".so"));
 		if(!m.lib->load())
 		{
 			MessageBox::msg(tr("Cannot load %1 module library.:\n%2").arg(module_name).arg(m.lib->error()));
