--- modules/default_sms/default_sms.cpp.orig	2005-05-03 18:41:49.000000000 +0200
+++ modules/default_sms/default_sms.cpp	2005-10-05 11:44:45.000000000 +0200
@@ -411,26 +414,31 @@
 	kdebugf2();
 }
 
+static bool activated;
+
 void SmsGatewaySlots::onCloseConfigDialog()
 {
 	kdebugf();
-	modules_manager->moduleDecUsageCount("default_sms");
+	if (activated)	
+		modules_manager->moduleDecUsageCount("default_sms"); 
+	activated=false;
 }
 
 void SmsGatewaySlots::onCreateConfigDialog()
 {
 	kdebugf();
 	
 	actualEraGateway=config_file.readEntry("SMS", "EraGateway");
 	
 	QLineEdit *e_erauser= ConfigDialog::getLineEdit("SMS", "User ID (48xxxxxxxxx)");
 	QLineEdit *e_erapassword= ConfigDialog::getLineEdit("SMS", "Password");
 	e_erapassword->setEchoMode(QLineEdit::Password);
-	
-	e_erapassword->setText(config_file.readEntry("SMS", "EraGateway_"+config_file.readEntry("SMS", "EraGateway")+"_Password"));
-	e_erauser->setText(config_file.readEntry("SMS", "EraGateway_"+config_file.readEntry("SMS", "EraGateway")+"_User"));
 
-	modules_manager->moduleIncUsageCount("default_sms");
+	e_erapassword->setText(config_file.readEntry("SMS", "EraGateway_"+config_file.readEntry("SMS", "EraGateway")+"_Password"));
+	e_erauser->setText(config_file.readEntry("SMS", "EraGateway_"+config_file.readEntry("SMS", "EraGateway")+"_User", "48"));
+	
+	modules_manager->moduleIncUsageCount("default_sms"); 
+	activated=true;
 	kdebugf2();
 }
 
