diff -urN kadu-0.2.1.org/kadu/adduser.cpp kadu-0.2.1/kadu/adduser.cpp
--- kadu-0.2.1.org/kadu/adduser.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/adduser.cpp	Fri Apr 26 14:37:13 2002
@@ -23,7 +23,7 @@
 extern gg_session sess;
 extern Kadu * kadu;
 
-Adduser::Adduser(QDialog* parent=0, const char *name=0) {
+Adduser::Adduser(QDialog* parent, const char *name) {
 
 this->resize(400,150);
 this->setCaption(i18n("Add user"));
diff -urN kadu-0.2.1.org/kadu/chat.cpp kadu-0.2.1/kadu/chat.cpp
--- kadu-0.2.1.org/kadu/chat.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/chat.cpp	Fri Apr 26 14:44:07 2002
@@ -29,7 +29,7 @@
 #include <qstring.h>
 #include <qfont.h>
 #include <qtoolbutton.h>
-#include <iostream.h>
+#include <iostream>
 
 #include <stdio.h>
 #include <stdlib.h>
@@ -173,7 +173,7 @@
 	}
 }
 
-CustomInput::CustomInput (Chat* parent=0, const char *name=0) : QMultiLineEdit (parent, name) {
+CustomInput::CustomInput (Chat* parent, const char *name) : QMultiLineEdit (parent, name) {
   tata = parent;
   }
 
@@ -225,7 +225,7 @@
 	if (additchar != 'W')
 		urlparse.append(additchar);
 	
-	cout << "KK Chat: extracted " << urlbase << " start=" << urloccur << " end=" << endurl << endl;
+	std::cout << "KK Chat: extracted " << urlbase << " start=" << urloccur << " end=" << endurl << endl;
 
 //	edit.replace(edit.mid(urloccur, endurl - urloccur), urlparse);
 	edit.replace(urloccur, endurl - urloccur, urlparse);
@@ -320,7 +320,7 @@
 }
 
 /* invoked from outside when new message arrives, this is the window to the world */
-int Chat::checkPresence(uin_t uin, unsigned char * msg, time_t time,int formats_count=0,struct gg_msg_format * formats=NULL) {
+int Chat::checkPresence(uin_t uin, unsigned char * msg, time_t time,int formats_count,struct gg_msg_format * formats) {
 	cp_to_iso(msg);
 	kadu->autoaway->stop();
 	kadu->autoaway->start(300000);
diff -urN kadu-0.2.1.org/kadu/config.cpp kadu-0.2.1/kadu/config.cpp
--- kadu-0.2.1.org/kadu/config.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/config.cpp	Fri Apr 26 14:23:17 2002
@@ -23,7 +23,7 @@
 extern struct config config;
 extern QArray<userlist> userlist;
 
-ConfigDialog::ConfigDialog(QWidget *parent = 0, const char *name= 0) : QTabDialog (parent, name) {
+ConfigDialog::ConfigDialog(QWidget *parent, const char *name) : QTabDialog (parent, name) {
 
 this->setWFlags(Qt::WDestructiveClose);
 setupTab1();
@@ -457,4 +457,4 @@
 		config.notifies.append(QString::number(UserToUin(e_notifies->text(i))));
 		}
 	
-	}
\ No newline at end of file
+	}
diff -urN kadu-0.2.1.org/kadu/dcc.cpp kadu-0.2.1/kadu/dcc.cpp
--- kadu-0.2.1.org/kadu/dcc.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/dcc.cpp	Fri Apr 26 14:47:38 2002
@@ -23,7 +23,7 @@
 extern Kadu * kadu;
 extern struct config config;
 
-DccGet::DccGet(struct gg_dcc * dcc_ptr = 0, QDialog *parent = 0, const char *name= 0) : QDialog (parent, name) {
+DccGet::DccGet(struct gg_dcc * dcc_ptr, QDialog *parent, const char *name) : QDialog (parent, name) {
 vbox1 = new QVBox(this);
 vbox1->setMargin(5);
 dccsock = dcc_ptr;
diff -urN kadu-0.2.1.org/kadu/history.cpp kadu-0.2.1/kadu/history.cpp
--- kadu-0.2.1.org/kadu/history.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/history.cpp	Fri Apr 26 14:41:39 2002
@@ -34,7 +34,7 @@
 #include <unistd.h>
 #include <qlayout.h>
 
-char *timestamp(time_t customtime = 0)
+char *timestamp(time_t customtime)
 {
 	static char buf[100];
 
@@ -71,7 +71,7 @@
 	return buf;
 }
 
-void appendHistory(unsigned int uin, unsigned char * msg, bool own, time_t time = 0) {
+void appendHistory(unsigned int uin, unsigned char * msg, bool own, time_t time) {
 	FILE *f;
 
 	char * path = getenv("HOME");
diff -urN kadu-0.2.1.org/kadu/kadu.cpp kadu-0.2.1/kadu/kadu.cpp
--- kadu-0.2.1.org/kadu/kadu.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/kadu.cpp	Fri Apr 26 14:45:53 2002
@@ -769,7 +769,7 @@
 }
 
 /* send a message, send me a message */
-Message::Message (const QString & nick, bool tchat=false, QDialog* parent , const char *name) : QDialog (parent, name) {
+Message::Message (const QString & nick, bool tchat, QDialog* parent , const char *name) : QDialog (parent, name) {
   QAccel *acc = new QAccel( this );
   acc->connectItem( acc->insertItem(Key_Return+CTRL), this, SLOT(commitSend()) );
 
@@ -1338,7 +1338,7 @@
 
 	}
 
-void Kadu::setAway(bool with_description = false) {
+void Kadu::setAway(bool with_description) {
 	i_am_busy = true;
 
 	if (with_description)
diff -urN kadu-0.2.1.org/kadu/network.cpp kadu-0.2.1/kadu/network.cpp
--- kadu-0.2.1.org/kadu/network.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/network.cpp	Fri Apr 26 14:39:56 2002
@@ -25,7 +25,7 @@
 extern Kadu * kadu;
 
 	/* progress bar, now isn't the name self-explanatory? :> */
-  Operation::Operation(const QString & labelText, const QString & cancelButtonText, int totalSteps, QWidget *parent = 0 )
+  Operation::Operation(const QString & labelText, const QString & cancelButtonText, int totalSteps, QWidget *parent )
 : QProgressDialog(labelText, cancelButtonText, totalSteps, parent)
   {
 			steps = 1;
@@ -37,7 +37,7 @@
 			this->setAutoClose(true);
 			this->setAutoReset(true);
 			if (kadu->autohammer) {
-				cout << "No focus" << endl;
+			    std::cout << "No focus" << endl;
 				this->showMinimized();
 				}
 
@@ -101,7 +101,7 @@
       t->stop();
   }
 
-ChooseDescription::ChooseDescription ( QWidget * parent=0, const char * name=0) :
+ChooseDescription::ChooseDescription ( QWidget * parent, const char * name) :
 		QDialog(parent, name, true) {
 this->setWFlags(Qt::WDestructiveClose);
 this->setCaption(i18n("Select description"));
diff -urN kadu-0.2.1.org/kadu/sms.cpp kadu-0.2.1/kadu/sms.cpp
--- kadu-0.2.1.org/kadu/sms.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/sms.cpp	Fri Apr 26 14:25:18 2002
@@ -21,7 +21,7 @@
 
 extern void sigchldHndl (int whatever);
 
-Sms::Sms (unsigned int gsmno,const QString & name = "sms", QDialog* parent) : QDialog (parent, name) {
+Sms::Sms (unsigned int gsmno,const QString & name, QDialog* parent) : QDialog (parent, name) {
 QGridLayout * grid = new QGridLayout(this,3,4,10,3);
 
 body = new QMultiLineEdit(this);
@@ -117,4 +117,4 @@
 snprintf(len, sizeof(len), "%d", body->text().length());
 smslen->setText(len);
 
-}
\ No newline at end of file
+}
diff -urN kadu-0.2.1.org/kadu/userbox.cpp kadu-0.2.1/kadu/userbox.cpp
--- kadu-0.2.1.org/kadu/userbox.cpp	Fri Apr 26 14:49:12 2002
+++ kadu-0.2.1/kadu/userbox.cpp	Fri Apr 26 14:27:12 2002
@@ -19,7 +19,7 @@
 
 extern QArray<userlist> userlist;
 
-MyListBox::MyListBox ( QWidget * parent=0, const char * name=0, WFlags f=0 ) : QListBox(parent, name),
+MyListBox::MyListBox ( QWidget * parent, const char * name, WFlags f ) : QListBox(parent, name),
 	QToolTip(viewport()) {
 
 }
@@ -85,4 +85,4 @@
 	else
 		return -1;
 
-	}
\ No newline at end of file
+	}
