--- kadu/modules/Makefile.am.orig	2005-10-14 16:08:01.000000000 +0200
+++ kadu/modules/Makefile.am	2005-10-14 16:09:16.000000000 +0200
@@ -68,9 +68,7 @@
 		if test -d translations; then \
 			cd translations && \
 			for trans in *.ts; do \
-				if [ ! -e $${trans//\ts/qm} ] || test $$trans -nt $${trans//\ts/qm}; then \
 					$(QT_LRELEASE_DIR)/lrelease $$trans; \
-				fi; \
 			done && \
 			cd ..; \
 		fi && \
