--- kadu/kadu/gadu.cpp.orig	2005-02-05 09:52:38.050863736 +0100
+++ kadu/kadu/gadu.cpp	2005-02-05 09:27:05.411860280 +0100
@@ -1250,7 +1250,7 @@
 		e.altnick = sections[3];
 		e.mobile = sections[4];
 		if (sections.count() >= 12)
-			groups = sections.count() - 11;
+			groups = sections.count() - 13;
 		else
 			groups = sections.count() - 7;
 		groupNames.clear();
--- kadu/kadu/userlist.cpp.orig	2005-02-05 09:52:26.515617360 +0100
+++ kadu/kadu/userlist.cpp	2005-02-05 09:27:05.408860736 +0100
@@ -488,7 +488,7 @@
 			userattribs = QStringList::split(";", line, TRUE);
 			kdebugm(KDEBUG_INFO, "UserList::readFromFile(): userattribs = %d\n", userattribs.count());
 			if (userattribs.count() >= 12)
-				groups = userattribs.count() - 11;
+				groups = userattribs.count() - 13;
 			else
 				groups = userattribs.count() - 7;
 			e.first_name = userattribs[0];
