--- kadu/kadu-core/CMakeLists.txt~	2010-03-22 23:42:23.000000000 +0100
+++ kadu/kadu-core/CMakeLists.txt	2011-03-10 09:31:24.977045288 +0100
@@ -169,7 +169,7 @@
 else (WIN32)
 	add_executable (kadu main.cpp main_unix.cpp ${kadu_SRCS} ${MOC_FILES})
 	set_target_properties (kadu PROPERTIES ENABLE_EXPORTS ON)
-	target_link_libraries (kadu ${STATIC_MODULES} ${QT_LIBRARIES} ${LIBGADU_LIBRARIES})
+	target_link_libraries (kadu ${STATIC_MODULES} ${QT_LIBRARIES} ${LIBGADU_LIBRARIES} ${X11_LIBRARIES} -ldl)
 endif (WIN32)
 
 install (TARGETS kadu DESTINATION bin)
--- kadu/CMakeLists.txt~	2010-09-22 17:42:42.000000000 +0200
+++ kadu/CMakeLists.txt	2011-03-10 09:32:09.820375268 +0100
@@ -147,6 +147,7 @@
 else (MINGW)
 
 pkg_search_module (LIBGADU REQUIRED libgadu>=${LIBGADU_MIN_VERSION})
+set (X11_LIBRARIES -lX11)
 
 execute_process (
 	COMMAND grep "${LIBGADU_MIN_VERSION}-rc1" "${LIBGADU_INCLUDEDIR}/libgadu.h"
