diff -urN kadu.org/kadu/config_file.cpp kadu/kadu/config_file.cpp
--- kadu.org/kadu/config_file.cpp	2003-08-09 23:44:56.310026416 +0200
+++ kadu/kadu/config_file.cpp	2003-08-09 23:45:33.962302400 +0200
@@ -45,7 +45,7 @@
 			else 
 				if (activegroup.name.length()) {
 					activeentry.name = line.section('=', 0, 0).stripWhiteSpace();
-					activeentry.value = line.section('=', 1, -1, QString::SectionIncludeTrailingSep).stripWhiteSpace();
+					activeentry.value = line.section('=', 1, -1, QString::SectionSkipEmpty);
 					if (line.contains('=') >= 1 && activeentry.name.length()
 						&& activeentry.value.length())
 						activegroup.entries.append(activeentry);
