--- kadu/modules/weather/translations/weather_pl.ts~	2008-01-22 20:28:27.000000000 +0100
+++ kadu/modules/weather/translations/weather_pl.ts	2009-05-01 19:24:00.185744711 +0200
@@ -96,10 +96,6 @@
         <translation>Powiadomienia</translation>
     </message>
     <message>
-        <source>Notifications</source>
-        <translation>Powiadomienia</translation>
-    </message>
-    <message>
         <source>Display</source>
         <translation>Wyświetlanie</translation>
     </message>
