--- kadu/kadu-core/CMakeLists.txt~	2009-10-11 14:41:36.000000000 +0200
+++ kadu/kadu-core/CMakeLists.txt	2010-11-08 12:37:46.349213000 +0100
@@ -163,7 +163,7 @@
 else (WIN32)
 	add_executable (kadu main.cpp main_unix.cpp ${kadu_SRCS} ${MOC_FILES})
 	set_target_properties (kadu PROPERTIES ENABLE_EXPORTS ON)
-	target_link_libraries (kadu ${STATIC_MODULES} ${QT_LIBRARIES} ${LIBGADU_LIBRARIES})
+	target_link_libraries (kadu ${STATIC_MODULES} ${QT_LIBRARIES} ${LIBGADU_LIBRARIES} ${X11_LIBRARIES})
 endif (WIN32)
 
 install (TARGETS kadu DESTINATION bin)
--- kadu/CMakeLists.txt~	2010-02-01 09:49:04.000000000 +0100
+++ kadu/CMakeLists.txt	2010-11-08 13:09:58.860517335 +0100
@@ -135,6 +135,7 @@
 else (MINGW)
 
 pkg_search_module (LIBGADU REQUIRED libgadu>=1.9.0)
+set (X11_LIBRARIES -lX11)
 
 execute_process (
 	COMMAND grep "1.9.0-rc1" ${LIBGADU_INCLUDEDIR}/libgadu.h
